invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/src/contrib/forms/Funding/CustomAwardForm.js

```diff
@@ -53,64 +55,73 @@ function CustomAwardForm({ deserializeFunder, selectedFunding }) {
-          if (selectedFundersArray.length === 1) {
-            const selectedFunder = selectedFundersArray[0];
-            if (selectedFunder) {
-              const deserializedFunder = serializeFunderFromDropdown(selectedFunder);
-              formikProps.form.setFieldValue(
-                "selectedFunding.funder",
-                deserializedFunder
-              );
+      <Overridable id="InvenioVocabularies.CustomAwardForm.RemoteSelectField.Container">
+        <RemoteSelectField
+          fieldPath="selectedFunding.funder.id"
+          suggestionAPIUrl="/api/funders"
+          suggestionAPIHeaders={{
+            Accept: "application/vnd.inveniordm.v1+json",
+          }}
+          placeholder={i18next.t("Search for a funder by name")}
+          serializeSuggestions={(funders) => {
+            return funders.map((funder) =>
+              deserializeFunderToDropdown(deserializeFunder(funder))
+            );
+          }}
+          searchInput={{
+            autoFocus: _isEmpty(selectedFunding),
+          }}
+          label={i18next.t("Funder")}
+          noQueryMessage={i18next.t("Search for funder...")}
+          clearable
+          allowAdditions={false}
+          multiple={false}
+          selectOnBlur={false}
+          selectOnNavigation={false}
+          required
+          search={(options) => options}
+          isFocused
+          onValueChange={({ formikProps }, selectedFundersArray) => {
+            if (selectedFundersArray.length === 1) {
+              const selectedFunder = selectedFundersArray[0];
+              if (selectedFunder) {
+                const deserializedFunder = serializeFunderFromDropdown(selectedFunder);
+                formikProps.form.setFieldValue(
+                  "selectedFunding.funder",
+                  deserializedFunder
+                );
+              }
             }
-          }
-        }}
-      />
-
-      <Header as="h3" size="small">
-        {i18next.t("Award information")} ({i18next.t("optional")})
-      </Header>
-      <Form.Group widths="equal">
-        <TextField
-          label={i18next.t("Number")}
-          placeholder={i18next.t("Award number")}
-          fieldPath="selectedFunding.award.number"
-        />
-        <TextField
-          label={i18next.t("Title")}
-          placeholder={i18next.t("Award Title")}
-          fieldPath="selectedFunding.award.title"
-        />
-        <TextField
-          label={i18next.t("URL")}
-          placeholder={i18next.t("Award URL")}
-          fieldPath="selectedFunding.award.url"
+          }}
         />
+      </Overridable>
+      <Overridable id="InvenioVocabularies.CustomAwardForm.AwardInformationHeader.Container">
+        <Header as="h3" size="small">
+          {i18next.t("Award information")} ({i18next.t("optional")})
+        </Header>
+      </Overridable>
+      <Form.Group widths="equal">
+        <Overridable id="InvenioVocabularies.CustomAwardForm.AwardNumberTextField.Container">
```

Comment:
small suggestion
```suggestion
        <Overridable id="InvenioVocabularies.CustomAwardForm.AwardNumberTextField.Container" fieldPath="selectedFunding.award.number">
```
if I remember correctly, this would prevent you from having to add fieldPath prop (and redefine it with the same path) on the component which will override this part - the path should be the same either way, otherwise the form won't work correctly